doc/reference/storage_ceph.md

```diff
@@ -109,8 +109,8 @@ Key                           | Type                          | Default
 
 Key                     | Type      | Condition                 | Default                                        | Description
 :--                     | :---      | :--------                 | :------                                        | :----------
-`block.filesystem`      | string    | block based driver        | same as `volume.block.filesystem`              | {{block_filesystem}}
-`block.mount_options`   | string    | block based driver        | same as `volume.block.mount_options`           | Mount options for block devices
+`block.filesystem`      | string    | block-based driver        | same as `volume.block.filesystem`              | {{block_filesystem}}
+`block.mount_options`   | string    | block-based driver        | same as `volume.block.mount_options`           | Mount options for block-backed file system volumes
 `security.shifted`      | bool      | custom volume             | same as `volume.security.shifted` or `false`   | {{enable_ID_shifting}}
 `security.unmapped`     | bool      | custom volume             | same as `volume.security.unmapped` or `false`  | Disable ID mapping for the volume
 `size`                  | string    | appropriate driver        | same as `volume.size`                          | Size/quota of the storage volume
```

Comment:
The `appropriate driver` condition isn't relevant now that these tables are driver specific. If a pool doesn't support `size` the whole row wouldn't be there.